google-ads/src/main/java/com/google/ads/googleads/lib/logging/Event.java

```diff
@@ -128,6 +134,29 @@ public String getResponseAsText() {
       return getResponse() == null ? null : getResponse().toString();
     }
 
+    /**
+     * Attempts to read the GoogleAdsFailure from the response, if present. This is a relatively
+     * expensive operation when the proto is present, so take care not to negatively impact
```

Comment:
Instead of:

> so take care not to negatively impact performance

which  could be interpreted as a note for the maintainers of this method, how about:

> so call this method judiciously.

Reply:
Done